libraries/Microsoft.Bot.Connector/Authentication/AuthenticationConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Bot.Connector.Authentication
+{
+    /// <summary>
+    /// General configuration settings for authentication.
+    /// </summary>
+    /// <remarks>
+    /// Note that this is explicitly a class and not an interface, 
+    /// since interfaces don't support default values, after the initial release any change would break backwards compatibility.
+    /// </remarks>
+    public class AuthenticationConfiguration
+    {
+        public string[] RequiredEndorsements { get; set; }
+    }
```

Comment:
= new string[];

Consider also List<string>, and having it default to an empty list. 

Both solutions eliminate the need for null checks in the general case.